constants/const.go

```diff
@@ -0,0 +1,27 @@
+package constants
```

Comment:
Could the constants from flags.go be here too? What sets them apart?

Reply:
Addressing this in the following commit, therefore I'm marking this as resolved.